File: Age_in_Days_Converter.java
```java
import java.util.Scanner;

public class AgeInDaysConverter {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter your age in years: ");
        int ageInYears = scanner.nextInt();
        int ageInDays = ageInYears * 365;
        System.out.println("Your age in days is: " + ageInDays);
    }
}
```

File: Age_in_Days_Converter.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        Console.Write("Enter your age in years: ");
        int years = Convert.ToInt32(Console.ReadLine());
        int days = years * 365;
        Console.WriteLine("Your age in days is: " + days);
        Console.ReadKey();
    }
}